idx/shortcodes/register-impress-shortcodes.php

```diff
@@ -558,8 +558,8 @@ public function property_carousel_shortcode( $atts = array() ) {
 		// All Instance Values are strings for shortcodes but not widgets.
 		$output .= '
             <script>
-            jQuery(document).ready(function( $ ){
-                $(".impress-listing-carousel-' . $display . '").owlCarousel({
+              window.onload = function(){
```

Comment:
Would add this via `window.addEventListener` instead if you're wanting to move away from jQuery's `document.ready` otherwise additional scrips may override this function.